src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Controllers/DataSetsControllerTests.cs

```diff
@@ -278,6 +291,8 @@ await TestApp.AddTestData<PublicDataDbContext>(context =>
                     Assert.Equal(indicatorMeta.DecimalPlaces, indicator.DecimalPlaces);
                 }
 
+                Assert.Equal(dataSetVersion.TimePeriodMetas.Count, content.Indicators.Count);
```

Comment:
I think there's a mistake here.

`content.Indicators.Count` should be `content.TimePeriods.Count` ?

Reply:
Yep d'oh, copy-pasta!